src/styles/blocks/main.scss

```
@@ -0,0 +1,132 @@
+.cards {
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(4, 200px);
+
+  gap: 46px 48px;
+  padding: 50px 40px;
```

```suggestion
  padding-top: 50px;
```
do u need horizontal padding here?why?
u are already used gap for it